pkg/cli/internal/config.go

```diff
@@ -0,0 +1,47 @@
+/*
+Copyright 2017 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package internal
+
+import (
+	"log"
+	"os"
+
+	"sigs.k8s.io/kubebuilder/internal/config"
+)
+
+const (
+	noticeColor = "\033[1;36m%s\033[0m"
+)
+
+// ConfiguredAndV1 returns true if the project is already configured.
+func Configured() bool {
+	_, err := config.Read()
+	return err == nil || os.IsExist(err)
+}
+
+// ConfiguredAndV1 returns true if the project is already configured and is v1.
+func ConfiguredAndV1() bool {
```

Comment:
Why we should care about the V1? 
V1 is deprecated and I think that indeed we could remove it from the project already.

Reply:
Also, `ConfiguredAndV1` shows hurting the cohesion